changelogs/unreleased/0000000001-typo_fix.yml

```diff
@@ -0,0 +1,13 @@
+---
+description: Fixed two typos in language.rst file of core docs
+issue-nr: N/A
```

Comment:
This will cause a type error when parsing the file. Currently you should just leave the field out if there's no related issue.
```suggestion
```